src/Module/Api/Twitter/Friendships/Incoming.php

```diff
@@ -86,6 +86,6 @@ protected function rawContent(array $request = [])
 
 		self::setLinkHeader();
 
-		System::jsonExit($return);
+		$this->response->exit('incoming', ['incoming' => $return]);
```

Comment:
```suggestion
		$this->response->exitWithJson($return);
```

Reply:
The Twitter endpoints should support XML.